darwin/lib/AutoUpdater.js

```diff
@@ -0,0 +1,48 @@
+const GhReleases = require('electron-gh-releases')
+const { dialog, app } = require('electron')
+
+const dialogOpts = {
+  type: 'info',
+  buttons: ['Restart', 'Later'],
+  title: 'Application Update',
+  message: 'New Version is Available!',
+  detail: 'Restart the application to apply the updates.'
```

Comment:
Maybe add a link to a changelog here?

Reply:
👍 I'll see if it's possible on the dialog